processor/metrics.go

```diff
@@ -155,21 +124,12 @@ func MetricViews(level telemetry.Level) []*view.View {
 		TagKeys:     tagKeys,
 		Aggregation: view.Sum(),
 	}
-	droppedSpansFromBadBatchesView := &view.View{
-		Name:        StatBadBatchDroppedSpanCount.Name(),
-		Measure:     StatBadBatchDroppedSpanCount,
-		Description: "The number of spans dropped from span batches with bad data.",
-		TagKeys:     tagKeys,
-		Aggregation: view.Sum(),
-	}
 
 	legacyViews := []*view.View{
 		receivedBatchesView,
 		droppedBatchesView,
 		receivedSpansView,
 		droppedSpansView,
-		droppedBadBatchesView,
```

Comment:
are these two views redundant?

Reply:
which 2? I am confused... The reason why I removed these 2 is because nobody records data for the StatBadBatchDroppedSpanCount measurement, so these are not going to produce data.

Reply:
ah, got it